src/char/pincode.c

```diff
@@ -249,6 +253,32 @@ void pincode_loginstate(int fd, struct char_session_data* sd, enum pincode_login
 	WFIFOSET(fd, 12);
 }
 
+// 0 = pin is correct
+// 1 = ask for pin - client sends 0x8b8
+// 2 = create new pin - client sends 0x8ba
+// 3 = pin must be changed - client 0x8be
+// 4 = create new pin ?? - client sends 0x8ba
+// 5 = client shows msgstr(1896)
+// 6 = client shows msgstr(1897) Unable to use your KSSN number
+// 7 = char select window shows a button - client sends 0x8c5
+// 8 = pincode was incorrect
+// [4144] pincode_loginstate2 can replace pincode_loginstate,
+// but kro using pincode_loginstate2 only for send wrong pin error or locked after 3 pins wrong
+void pincode_loginstate2(int fd, struct char_session_data* sd, enum pincode_login_response state, enum pincode_login_response2 flag)
```

Comment:
Packet is CZ or ZC ?

Reply:
not CZ and not ZC. this is char server...

Reply:
isn't it ZC? because it sends from server to client

Reply:
Maybe PACKET_HC_SECOND_PASSWD_LOGIN2 ?

Reply:
yes this HC packet. it totally same with old one. But for some reason kro using new packet only for message with wrong pin and blocked pin for 3 days (hercules cant do this for now)